core/src/main/scala/kafka/admin/AclCommand.scala

```diff
@@ -315,9 +315,9 @@ object AclCommand extends Logging {
           println(s"Adding ACLs for resource `$resource`: $Newline ${acls.map("\t" + _).mkString(Newline)} $Newline")
           val aclBindings = acls.map(acl => new AclBinding(resource, acl))
           authorizer.createAcls(null, aclBindings.toList.asJava).asScala.foreach { result =>
-            if (result.failed) {
-              println(s"Error while adding ACLs: ${result.exception.getMessage}")
-              println(Utils.stackTrace(result.exception))
+            result.exception.ifPresent { exception =>
```

Comment:
I think this doesn't work with Scala 2.11. You probably have to convert to the Scala option before you can use lambdas. There are utility methods for that in `scala-java8-compat`.

Reply:
@ijuma Thank you, updated.